src/w3c/linter-rules/required-sections.js

```diff
@@ -40,15 +55,20 @@ export function run(conf) {
     return;
   }
 
+  // We can't check for headers unless we also have a translation
+  if (!localizationStrings[lang]) {
+    return;
```

Comment:
Should add a warning here. A `console.warn` if not `showWarning`..

Reply:
Not sure about that ... there is nothing the spec Editor can do about it? 

Also, spec translations are not canonical, so the warnings is just nice to have in this context.

Reply:
Went with the  console.warn ... I think that works for now.

Reply:
Yeah, console.warn is good. As the author (or us) will know what's happening if that happens. Less unexpected.